lib/msf/core/exploit/remote/ms_icpr.rb

```diff
@@ -195,6 +196,19 @@ def do_request_cert(icpr, opts)
 
     return unless response[:certificate]
 
+    policy_oids = get_cert_policy_oids(response[:certificate])
+    if application_policies.present? && !(application_policies - policy_oids.map(&:value)).empty?
+      print_error('Certificate application policy OIDs were submitted, but some are missing in the response. This indicates the target has received the patch for ESC15 (CVE-2024-49019).')
```

Comment:
This error will be printed if you attempt to run the module with `ADD_CERT_APP_POLICY` against a template that isn't vulnerable to ESC15. So we should update the message accordingly. 
```suggestion
      print_error('Certificate application policy OIDs were submitted, but some are missing in the response. This indicates the target has received the patch for ESC15 (CVE-2024-49019) or the template is not vulnerable.')
```